net/unbound/Portfile

```diff
@@ -74,8 +75,20 @@ startupitem.create	yes
 startupitem.name	unbound
 startupitem.logfile	/Library/Logs/unbound-startupitem.log
 startupitem.logevents	yes
-startupitem.start	"(${prefix}/sbin/unbound-anchor -a ${prefix}/etc/${name}/root.key 2>&1) || : && (chown ${unbounduser}:${unboundgroup} ${prefix}/etc/${name}/root.key 2>&1) && (${prefix}/sbin/unbound 2>&1)"
-startupitem.stop	"(/bin/kill \$(cat ${prefix}/var/run/${name}/unbound.pid) 2>&1)"
+# the following entries handle multiple instances running in parallel
+startupitem.start	"(${prefix}/sbin/unbound-anchor -a ${prefix}/etc/${name}/root.key 2>&1) || : && (chown ${unbounduser}:${unboundgroup} ${prefix}/etc/${name}/root.key 2>&1) && (for i in ${prefix}/etc/${name}/${name}*conf; do ${prefix}/sbin/unbound -c \$i 2>&1; done)"
```

Comment:
Actually, `unbound -c $i` needs the `$i` quoted. If someone's prefix has a space in the path, it'll break this script.

Reply:
Basically every variable used that's non-numeric: all paths and words need to be in a quoted string to avoid word-splitting.

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2231